lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -360,6 +405,25 @@ def get_preview_body(self, obj):
         """
         return Truncator(strip_tags(self.get_rendered_body(obj))).chars(35, ).replace('\n', ' ')
 
+    def get_close_reason(self, obj):
+        """
+        Returns the reason for which the thread was closed.
+        """
+        if not _validate_privileged_access(self.context):
+            return None
+        reason_code = obj.get("close_reason_code")
+        if not reason_code:
+            return None
```

Comment:
@xitij2000 I think these two lines are redundant. `CLOSED_REASON_CODES.get(None)` would return `None`, so this if check is not needed.